app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
+  def create
+    comment = Comment.new(comment_params)
+    comment.user_id = current_user.id
+    if comment.save
+      flash.now[:success] = 'Comment created'
+    else
+      flash.now[:danger] = 'Some errors'
+    end
+    redirect_to user_path(current_user)
+  end
+
```

Comment:
[OPTIONAL]
- Create an edit method for the Comments controller. This way you can implement an edit functionality. This change can add relevant value to your page since it will make more similar to real-world web apps.
- It would be also great if you create a destroy method. It makes a lot of sense for a user to have an option to delete his own comments.

Reply:
Thanks a lot, we'll do that at the end of the project.